tests/idsvr-docker.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load 'bats-support/load'
+load 'bats-assert/load'
+
+@test "Check that idsvr user can access idsvr content inside container" {
+    run docker run -i $BATS_CURITY_IMAGE bash <<EOF
```

Comment:
do we want the `-t` flag?

Reply:
no, we dont need. When I add that I got `the input device is not a TTY` because we are using a temporary container (using the image instead of container ID here). So this will work fine